.travis.yml

```diff
@@ -21,9 +20,9 @@ matrix:
 addons:
   apt:
     packages:
-      llvm-3.9
-      llvm-3.9-tools
-      libclang-3.9-dev
+      llvm-6.0
```

Comment:
Looks like these aren't available in the travis image.

Reply:
The ultimate solution would be BinaryBuilder. I tried LLVMBuilder yesterday, but also hit     https://github.com/staticfloat/LLVMBuilder/issues/33 . Then, I tried LLVM official binaries and it worked well. However, LLVM doesn't officially ship generic Linux binaries, so we still need to find a way to work around LLVMBuilder#33.